client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -72,6 +74,17 @@ function EmailPlanMailboxesList( { account, domain, isLoadingEmails, mailboxes }
 	const translate = useTranslate();
 	const accountType = account?.account_type;
 
+	let emptyText = translate( 'No mailboxes' );
+
+	if (
+		isRecentlyRegistered( domain.registrationDate, 45 ) &&
+		getGSuiteSubscriptionStatus( domain ) === 'unknown'
+	) {
+		emptyText = translate(
+			'We are configuring your mailboxes. You will receive an email shortly when they are ready to use.'
+		);
+	}
+
```

Comment:
I think we could move that block of code closer to where the `emptyText` variable is actually used. We could even move it inside the corresponding `if` like this:

```javascript
if ( ! mailboxes || mailboxes.length < 1 ) {
  let text = translate( 'No mailboxes' );

  if (
    isRecentlyRegistered( domain.registrationDate, 45 ) &&
    getGSuiteSubscriptionStatus( domain ) === 'unknown'
  ) {
    text = translate(
      'We are configuring your mailboxes. You will receive an email shortly when they are ready to use.'
    );
  }
  
  return (
    <MailboxListHeader accountType={ accountType }>
      <MailboxListItem hasNoEmails>
        <span>{ text }</span>
      </MailboxListItem>
    </MailboxListHeader>
  );
}
```

Reply:
Sure, that certainly improves readability!